modules/administration/pages/content-lifecycle.adoc

```diff
@@ -37,49 +35,54 @@ The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
 [NOTE]
-.Content Filters
 ====
-Content Filters are global in your organization and can be shared in multiple projects.
+Content filters are global in your organization and can be shared between projects.
 ====
 
 [NOTE]
-.Adding environment in existing project
 ====
-When you add an environment to an existing project that has already some sources built, the newly added environment will be populated with the content of its predecessor environment.
+If your project already contains built sources, when you add an environment it will automatically populate with the existing content.
```

Comment:
I think the "predecessor" part is important to be mentioned. I like the rephrase but we should still say that. Maybe something like:
`If your project already contains built sources, when you add an environment it will automatically populate with the existing content from the previous environment of the cycle or, if the new environment takes the first place, it will get the latest sources built version.`